meinberlin/assets/scss/components/_item_detail.scss

```diff
@@ -39,8 +39,7 @@
     margin-bottom: 1.5em;
 }
 
-.item-detail__image,
-.item-detail__image--background {
+.item-detail__image {
```

Comment:
I know, you didn't add that, but when we are on it, can we change the naming? `item-detail__image--background` is not a variant, so maybe we call it `item-detail__background-image` and the other one `item-detail__hero-image`? Or only the first, because less changes.